stable_baselines3/common/policies.py

```diff
@@ -171,6 +172,10 @@ def load(cls, path: str, device: Union[th.device, str] = "auto") -> "BaseModel":
         """
         device = get_device(device)
         saved_variables = th.load(path, map_location=device)
+
+        if "sde_net_arch" in saved_variables["data"]:
```

Comment:
Maybe a warning here that the parameter is now removed and to use it one has to downgrade SB3?